src/inlineScriptTags.js

```diff
@@ -4,17 +4,20 @@ const fs = require('fs').promises;
 const path = require('path');
 
 let inlineHtmlScripts = async htmlPath => {
-	const scriptTagRegex = /<script src="([\w.\-\/]+)"><\/script>/;
+	const scriptTagRegex = /<script .*src="([\w.\-\/]+)".*><\/script>/;
```

Comment:
Let's replace the first `.*` with `(?:.* )?` in order to avoid incorrectly parsing e.g. `<script  notsrc="bad-src" src="good-src">`

```suggestion
	const scriptTagRegex = /<script (?:.* )?src="([\w.\-\/]+)".*><\/script>/;
```

Reply:
Changed